app-crypt/steghide/steghide-0.5.1.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A Steganography Program"
+DESCRIPTION="Steghide is a steganography program that \
+is able to hide data in various kinds of image and \
+audio files. The color respectivly sample-frequencies \
+are not changed thus making the embedding resistant \
+against first-order statistical tests."
+HOMEPAGE="http://steghide.sourceforge.net/"
+COPYRIGHT="2001-2003 Stefan Hetzl"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/steghide/steghide-$portVersion.tar.gz"
+CHECKSUM_SHA256="78069b7cfe9d1f5348ae43f918f06f91d783c2b3ff25af021e6a312cf541b47b"
+PATCHES="steghide-0.5.1.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	steghide = $portVersion
+	cmd:steghide = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libmhash
+	lib:libmcrypt
+	lib:libjpeg
+	lib:libz
+	lib:libintl
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libmhash
+	devel:libmcrypt
+	devel:libjpeg
+	devel:libz
+	devel:libintl
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:gcc
+	cmd:gettext
+	cmd:libtoolize
+	cmd:make
+	"
+
+BUILD()
```

Comment:
When I run haikuporter with `--strict-policy` (and I always do so, with the shorter `-S` option) it buidls a package but shows this warning:
```
Warning: POLICY ERROR: Invalid top-level package entry "share"
Warning: Error: packaging policy violation(s) in strict mode
Error: Build has failed - stopping.
```
So I just need to go in `app-crypt/steghide/work-0.5.1/packaging/steghide/share` and I see that the Makefile is installing several files in `doc/steghide`: ABOUT-NLS BUGS COPYING CREDITS HISTORY INSTALL LEAME README TODO

This comes from this line in `Makefile.am`
```
grep ^docdir Makefile.am 
docdir = $(prefix)/share/doc/$(PACKAGE)
```
(On Haiku we don't have this share directory.)

When I get rid of that line, those files go to the right directory: `documentation/packages/steghide` (instead of `share/doc/steghide`)

We can add this in a PATCH() function.

Reply:
Or you could just move it over in `INSTALL()`

Reply:
No, it's better, imo, to fix this earlier. It works fine with this:
```
PATCH()
{
	sed -i -e "/^docdir =/ d;" Makefile.am
}
```